examples/debian_snapshot/bullseye.yaml

```diff
@@ -11,7 +11,9 @@ version: 1
 
 sources:
   - channel: bullseye main contrib
-    url: https://snapshot-cloudflare.debian.org/archive/debian/20240210T223313Z
+    urls:
```

Comment:
Do you want to run lock for this?

Reply:
I pushed the commit with the "old locks" so you could run the CI with them being old & the new code, and then the CI with both new code and new locks :)  But it's OK, I did test both by hand locally, plus I think the code is clear enough that both cases do work.